subprojects/ide-native/src/integTest/groovy/org/gradle/ide/xcode/XcodeMultipleCppProjectIntegrationTest.groovy

```diff
@@ -84,7 +84,7 @@ class XcodeMultipleCppProjectIntegrationTest extends AbstractXcodeIntegrationSpe
             .succeeds()
 
         then:
-        resultReleaseApp.assertTasksExecuted(':greeter:dependReleaseCpp', ':greeter:compileReleaseCpp', ':greeter:linkRelease',
+        resultReleaseApp.assertTasksExecuted(':greeter:dependReleaseCpp', ':greeter:compileReleaseCpp', ':greeter:linkRelease', ':greeter:extractSymbolsRelease', ':greeter:stripSymbolsRelease',
```

Comment:
So, per our conversation, the executable that we point Xcode at is the unstripped binary with debug symbols (this is what Xcode will point at, too, when Gradle is not involved).  The upstream libraries that it links against are the stripped binaries of those libs (which is where these additional tasks are coming from).  This is because we set the link time file for those libraries to be the stripped file (as opposed to the unstripped file).  See the discussion of this at https://github.com/gradle/gradle-native/issues/214#issuecomment-346460621.  I don't think this needs to change unless we decide we should change the link time file to be the unstripped binary.